common/locales/en/questsContent.json

```diff
@@ -89,12 +89,12 @@
   "questMoonstone1Text": "Recidivate, Part 1: The Moonstone Chain",
   "questMoonstone1Notes": "A terrible affliction has struck Habiticans. Bad Habits thought long-dead are rising back up with a vengeance. Dishes lie unwashed, textbooks linger unread, and procrastination runs rampant!<br><br>You track some of your own returning Bad Habits to the Swamps of Stagnation and discover the culprit: the ghostly Necromancer, Recidivate. You rush in, weapons swinging, but they slide through her specter uselessly.<br><br>\"Don’t bother,\" she hisses with a dry rasp. \"Without a chain of moonstones, nothing can harm me – and master jeweler @aurakami scattered all the moonstones across Habitica long ago!\" Panting, you retreat... but you know what you must do.",
   "questMoonstone1CollectMoonstone": "Moonstones",
-  "questMoonstone1DropMoonstone2Quest": "The Moonstone Chain Part 2: Recidivate the Necromancer (Scroll)",
+  "questMoonstone1DropMoonstone2Quest": "Recidivate, Part 2: Recidivate The Necromancer (Scroll)",
```

Comment:
I'm ok with changing it but the reason it's capitalized here is because it's also capitalized in the name below. If we change it here, we should change it there too.